examples/OrthogonalRegressionNonAnalytic_no-doc.ipynb

```diff
@@ -4,7 +4,28 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Example that shows how the `use_orthogonal_projector` option in `skcosmo.linear_model.OrthogonalRegression` can result in nonanalytic behavior of the prediction with respect to changes in the feature dimension."
+    "Regression with orthogonal projector/matrices\n",
+    "================================================================"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this example we explain how the `use_orthogonal_projector` option in `skcosmo.linear_model.OrthogonalRegression` can result in nonanalytic behavior for predictions with respect to changes in the feature dimension.\n",
+    "`skcosmo.linear_model.OrthogonalRegression` solves the linear regression problem\n",
+    "\n",
+    "$$ \\min_\\Omega ||y - X\\Omega\\||_F$$\n",
+    "\n",
+    "with the constraint that $\\Omega$ is an orthogonal projection/matrix  $\\Omega^T\\Omega=I$. In this formulation it can be only solved if $y$ and $X$ have the same dimension, i.e. they agree in the number of features. If `use_orthogonal_projector` is set to False, the feature dimension of $X$ or $y$ are padded with 0 values such that they agree in the number of dimension. To be able to compute an orthogonal projector without any padding of the matrices the option `use_orthogonal_projector` is offered with the caveat that it does not always well behave analytically. For this approach we take the linear regression solution $W$\n",
+    "\n",
+    "$$ \\min_W ||y - XW\\||_F,$$\n",
+    "\n",
+    "applyt SVD on it $W = USV^T$ and solve the Procrustes problem for\n",
+    "\n",
+    "$$\\min_\\Omega' ||yV - XU\\Omega'\\||_F\\quad \\Omega'^T\\Omega'=I$$\n",
+    "\n",
+    "The final orthogonal projector is then $\\Omega =  U\\Omega' V^T$. This solution can however be problematic, since changing the number of features can result in different predictions. This notebook shows an example of this problematic behaviour. "
```

Comment:
This set of sentences doesn't make much sense. From a simple copy-editing standpoint, it should read:

> In this example, we explain how when using `skcosmo.linear_model.OrthogonalRegression` the option `use_orthogonal_projector` can result in non-analytic behavior. 
> In `skcosmo.linear_model.OrthogonalRegression`, we solve the linear regression problem assuming an orthogonal weighting matrix $\\Omega$ to project from the feature space $X$ to the target space $y$.
> $$\\min_\\Omega ||y - X\\Omega\\||_F$$
> This assumes that $X$ and $y$ contain the same number of features. 
> If `use_orthogonal_projector=False`, the smaller of $X$ and $y$ is padded with null features, i.e. columns of zeros. 
> However, when `use_orthogonal_projector=True`, we begin with the weights $W$ determined by the linear regression problem
> $$ \\min_W ||y - XW\\||_F,$$
> and solve the Procrusted problem for 
> $$\\min_\\Omega' ||yV - XU\\Omega'\\||_F\\quad \\Omega'^T\\Omega'=I$$
> where the SVD of $W = USV^T$
> The final orthogonal projector is then $\\Omega =  U\\Omega' V^T$. 
> In this notebook, we demonstrate a problem that may arise with this solution, as changing the number of features can result in different predictions.